scripts/cert_generation.sh

```diff
@@ -265,3 +299,18 @@ generate_certs 'system.admin'
 generate_certs 'kibana-internal' "$(generate_extensions false false kibana)"
 generate_certs 'elasticsearch' "$(generate_extensions true true $LOG_STORE{,-cluster}{,.${NAMESPACE}.svc}{,.cluster.local})"
 generate_certs 'logging-es' "$(generate_extensions false true $LOG_STORE{,.${NAMESPACE}.svc}{,.cluster.local})"
+
+if [ ! -s "${WORKING_DIR}/kibana-session-secret" ] ; then
+  info "Generating kibana session secret"
+  result=$(python -c "import string,random;print(''.join(random.choice(string.ascii_letters + string.digits) for i in range(32)))")
```

Comment:
This achieves almost the same end, with a shorter alphabet of letters, without the Python dependency
```
[root@e33867c18033 cluster-logging-operator]# dd if=/dev/urandom count=1 ibs=16 status=none | hexdump -e '"%02X"'
4306BCE91BC0C99B7AB482ECC3E73047
```

Reply:
updated to remove python dep